observation_portal/requestgroups/filters.py

```diff
@@ -31,12 +31,13 @@ class RequestGroupFilter(django_filters.FilterSet):
             'modified': 'Last Update'
         }
     )
+    request_id = django_filters.CharFilter(field_name='requests__id')
```

Comment:
should this be an integer filter instead of char filter - I think the ids are autofields, which are integer fields...

Reply:
Hmm yeah maybe... we do the same thing in the [observation filters](https://github.com/observatorycontrolsystem/observation-portal/blob/master/observation_portal/observations/filters.py#L48). Do you happen to remember why we went with a charfield there?

Reply:
nope, I see I did that, but I probably just didn't know and it worked? It should probably be an integer filter too.

Reply:
Ok I'll change them